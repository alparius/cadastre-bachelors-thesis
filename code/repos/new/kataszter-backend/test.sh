#!/bin/bash

rm -f coverage.txt
echo 'mode: atomic' > coverage.txt

for d in $(go list ./... | grep -v testing); do
    go test -v -p 1 -race -covermode=atomic -coverprofile=coverage.out -coverpkg=$(go list ./...| grep -v testing | tr "\n" ",") $d
    if [ -f coverage.out ]; then
        tail -n +2 coverage.out >> coverage.txt || exit 255
        rm coverage.out
    fi
done
