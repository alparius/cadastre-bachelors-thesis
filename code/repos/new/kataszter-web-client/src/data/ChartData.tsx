interface IChartData {
    key: any;
    data: any;
    counter: number;
}

export default IChartData;
