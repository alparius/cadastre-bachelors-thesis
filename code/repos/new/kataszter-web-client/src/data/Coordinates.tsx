interface ICoordinates {
    lat: number;
    lng: number;
}

export let NewCoordinates: ICoordinates = {
    lat: 0,
    lng: 0
};

export default ICoordinates;
