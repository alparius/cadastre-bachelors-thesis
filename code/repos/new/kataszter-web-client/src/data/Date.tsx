interface IDate {
    year: number;
    month: number;
    day: number;
    comment: string;
}

export let NewDate: IDate = {
    year: 0,
    month: 0,
    day: 0,
    comment: ""
};

export default IDate;
