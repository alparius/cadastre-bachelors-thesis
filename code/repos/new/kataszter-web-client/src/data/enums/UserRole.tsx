enum UserRole {
    OWNER = "owner",
    ADMIN = "admin",
    EDITOR = "editor",
    READER = "reader",
    GUEST = "guest",
    PENDING = "pending",
    REFUSED = "refused"
}

export default UserRole;
