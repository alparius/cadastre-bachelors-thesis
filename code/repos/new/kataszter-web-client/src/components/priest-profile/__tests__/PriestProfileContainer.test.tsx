/*===================
  PriestProfileContainer.unit.test.tsx
 ===================*/
import React from "react";

import UserRole from "../../../data/enums/UserRole";
import IPriestProfileData from "../../../data/priest/PriestProfileData";
import { NewTestPriestProfileData } from "../../../util/test/data/test-data";
import { createComponentWithIntlAndRouter, mountWithIntl } from "../../../util/test/test-wrapper";
import PriestProfileContainer from "../PriestProfileContainer";

const handlers = {
    setPriest: jest.fn(),
    setEditMode: jest.fn(),
    setFiles: jest.fn(),
    setFileType: jest.fn(),
    asyncPostFile: jest.fn()
};

const priest: IPriestProfileData = NewTestPriestProfileData;

describe("PriestProfileContainer element", () => {
    afterEach(() => {
        handlers.setPriest.mockClear();
        handlers.setEditMode.mockClear();
        handlers.setFiles.mockClear();
        handlers.setFileType.mockClear();
        handlers.asyncPostFile.mockClear();
    });

    it("should render a profile container when the priest is not empty", () => {
        const container = createComponentWithIntlAndRouter(
            <PriestProfileContainer
                responsePostFile={undefined}
                currentTab={"general"}
                priest={priest}
                loading={false}
                showError={false}
                userRole={undefined}
                editMode={false}
                errorStatusPostFile={200}
                {...handlers}
            />
        ).toJSON();
        expect(container).toMatchSnapshot();
    });

    it("should render a placeholder when the priest is empty", () => {
        const emptyPriestProfile: IPriestProfileData[] = [];
        const container = createComponentWithIntlAndRouter(
            <PriestProfileContainer
                responsePostFile={undefined}
                currentTab={"general"}
                priest={emptyPriestProfile[0]}
                loading={false}
                showError={false}
                userRole={undefined}
                editMode={false}
                errorStatusPostFile={200}
                {...handlers}
            />
        ).toJSON();
        expect(container).toMatchSnapshot();
    });

    it("should render an error message when showError is true", () => {
        const container = createComponentWithIntlAndRouter(
            <PriestProfileContainer
                responsePostFile={undefined}
                currentTab={"general"}
                priest={priest}
                loading={false}
                showError
                userRole={undefined}
                editMode={false}
                errorStatusPostFile={200}
                {...handlers}
            />
        ).toJSON();
        expect(container).toMatchSnapshot();
    });

    it("should render an error message when showError and loading are both true", () => {
        const container = createComponentWithIntlAndRouter(
            <PriestProfileContainer
                responsePostFile={undefined}
                currentTab={"general"}
                priest={priest}
                loading
                showError
                errorStatusPostFile={200}
                userRole={undefined}
                editMode={false}
                {...handlers}
            />
        ).toJSON();
        expect(container).toMatchSnapshot();
    });

    it("should render a placeholder card when only loading is true", () => {
        const container = createComponentWithIntlAndRouter(
            <PriestProfileContainer
                responsePostFile={undefined}
                currentTab={"general"}
                priest={priest}
                loading
                showError={false}
                errorStatusPostFile={200}
                userRole={undefined}
                editMode={false}
                {...handlers}
            />
        ).toJSON();
        expect(container).toMatchSnapshot();
    });

    //
    it("should render a profile container when the priest is not empty while editMode=true", () => {
        const container = createComponentWithIntlAndRouter(
            <PriestProfileContainer
                responsePostFile={undefined}
                currentTab={"general"}
                priest={priest}
                loading={false}
                showError={false}
                errorStatusPostFile={200}
                userRole={undefined}
                editMode
                {...handlers}
            />
        ).toJSON();
        expect(container).toMatchSnapshot();
    });

    it("should render a placeholder when the priest is empty while editMode=true", () => {
        const emptyPriestProfile: IPriestProfileData[] = [];
        const container = createComponentWithIntlAndRouter(
            <PriestProfileContainer
                responsePostFile={undefined}
                currentTab={"general"}
                priest={emptyPriestProfile[0]}
                loading={false}
                showError={false}
                errorStatusPostFile={200}
                userRole={undefined}
                editMode
                {...handlers}
            />
        ).toJSON();
        expect(container).toMatchSnapshot();
    });

    it("should render an error message when showError is true and editMode=true", () => {
        const container = createComponentWithIntlAndRouter(
            <PriestProfileContainer
                responsePostFile={undefined}
                currentTab={"general"}
                priest={priest}
                loading={false}
                showError
                errorStatusPostFile={200}
                userRole={undefined}
                editMode
                {...handlers}
            />
        ).toJSON();
        expect(container).toMatchSnapshot();
    });

    it("should call setEditMode if edit Button pressed", () => {
        const container = mountWithIntl(
            <PriestProfileContainer
                responsePostFile={undefined}
                currentTab={"general"}
                priest={priest}
                loading={false}
                showError={false}
                errorStatusPostFile={200}
                userRole={UserRole.ADMIN}
                editMode={false}
                {...handlers}
            />
        );
        container.find("#editModeButton").at(0).simulate("click");
        expect(handlers.setEditMode).toHaveBeenCalledTimes(1);
    });

    it("should call setPriest, setEditMode if saveChangeButton pressed", () => {
        const container = mountWithIntl(
            <PriestProfileContainer
                responsePostFile={undefined}
                currentTab={"general"}
                priest={priest}
                loading={false}
                showError={false}
                errorStatusPostFile={200}
                userRole={UserRole.ADMIN}
                editMode
                {...handlers}
            />
        );

        container.find("#saveChanges").at(0).simulate("click");
        expect(handlers.setPriest).toHaveBeenCalledTimes(1);
        expect(handlers.setEditMode).toHaveBeenCalledTimes(1);
    });

    it("should call setEditMode if revertChangesButton pressed", () => {
        const container = mountWithIntl(
            <PriestProfileContainer
                responsePostFile={undefined}
                currentTab={"general"}
                priest={priest}
                loading={false}
                showError={false}
                errorStatusPostFile={200}
                userRole={UserRole.ADMIN}
                editMode
                {...handlers}
            />
        );

        container.find("#revertChanges").at(0).simulate("click");
        expect(handlers.setEditMode).toHaveBeenCalledTimes(1);
    });
});
