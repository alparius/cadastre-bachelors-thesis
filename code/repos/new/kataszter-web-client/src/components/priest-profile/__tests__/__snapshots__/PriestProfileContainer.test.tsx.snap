// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PriestProfileContainer element should render a placeholder card when only loading is true 1`] = `
<div
  className="ui fluid card"
  onClick={[Function]}
>
  <div
    className="content"
  >
    <div
      className="header"
    >
      <div
        className="ui placeholder"
      >
        <div
          className="header"
        >
          <div
            className="line"
          />
        </div>
      </div>
    </div>
    <div
      className="description"
    >
      <div
        className="ui placeholder"
      >
        <div
          className="line"
        />
        <div
          className="line"
        />
        <div
          className="line"
        />
        <div
          className="line"
        />
        <div
          className="line"
        />
        <div
          className="line"
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`PriestProfileContainer element should render a placeholder when the priest is empty 1`] = `
<div
  className="ui fluid card"
  onClick={[Function]}
>
  <div
    className="content"
  >
    <div
      className="header"
    >
      <div
        className="ui placeholder"
      >
        <div
          className="header"
        >
          <div
            className="line"
          />
        </div>
      </div>
    </div>
    <div
      className="description"
    >
      <div
        className="ui placeholder"
      >
        <div
          className="line"
        />
        <div
          className="line"
        />
        <div
          className="line"
        />
        <div
          className="line"
        />
        <div
          className="line"
        />
        <div
          className="line"
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`PriestProfileContainer element should render a placeholder when the priest is empty while editMode=true 1`] = `
<div
  className="ui fluid card"
  onClick={[Function]}
>
  <div
    className="content"
  >
    <div
      className="header"
    >
      <div
        className="ui placeholder"
      >
        <div
          className="header"
        >
          <div
            className="line"
          />
        </div>
      </div>
    </div>
    <div
      className="description"
    >
      <div
        className="ui placeholder"
      >
        <div
          className="line"
        />
        <div
          className="line"
        />
        <div
          className="line"
        />
        <div
          className="line"
        />
        <div
          className="line"
        />
        <div
          className="line"
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`PriestProfileContainer element should render a profile container when the priest is not empty 1`] = `
Array [
  <div
    className="ui four column grid"
  >
    <div
      className="column"
    />
    <div
      className="column"
    >
      <div
        className="ui placeholder"
      >
        <div
          className="rectangular image"
        />
      </div>
    </div>
    <div
      className="column"
    >
      <div
        className="ui placeholder"
      >
        <div
          className="line"
        />
      </div>
    </div>
    <div
      className="column"
    />
  </div>,
  <div
    className="ui divider"
  />,
  <div
    className="ui four column grid"
  >
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
  </div>,
  <div
    className="ui divider"
  />,
  <div
    className="ui four column grid"
  >
    <div
      className="column"
    />
    <div
      className="column"
    >
      <div
        className="ui placeholder"
      >
        <div
          className="rectangular image"
        />
      </div>
    </div>
    <div
      className="column"
    >
      <div
        className="ui placeholder"
      >
        <div
          className="line"
        />
      </div>
    </div>
    <div
      className="column"
    />
  </div>,
  <div
    className="ui divider"
  />,
  <div
    className="ui four column grid"
  >
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
  </div>,
]
`;

exports[`PriestProfileContainer element should render a profile container when the priest is not empty while editMode=true 1`] = `
Array [
  <div
    className="ui tertiary segment"
    id="editControlLabelComputer"
  >
    <div
      className="ui stackable two column grid"
    >
      <div
        className="middle aligned column"
        id="editModeText"
      >
        You are in edit mode.
      </div>
      <div
        className="column"
      >
        <div
          className="ui two column grid"
        >
          <div
            className="column"
          >
            <button
              className="ui violet fluid button"
              id="saveChanges"
              onClick={[Function]}
            >
              Save Changes
            </button>
          </div>
          <div
            className="column"
          >
            <button
              className="ui grey fluid button"
              id="revertChanges"
              onClick={[Function]}
              value={false}
            >
              Cancel
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="ui divider"
    style={
      Object {
        "marginTop": "4em",
      }
    }
  />,
  <div
    className="ui four column grid"
  >
    <div
      className="column"
    />
    <div
      className="column"
    >
      <div
        className="ui placeholder"
      >
        <div
          className="rectangular image"
        />
      </div>
    </div>
    <div
      className="column"
    >
      <div
        className="ui placeholder"
      >
        <div
          className="line"
        />
      </div>
    </div>
    <div
      className="column"
    />
  </div>,
  <div
    className="ui divider"
  />,
  <div
    className="ui four column grid"
  >
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
  </div>,
  <div
    className="ui divider"
  />,
  <div
    className="ui four column grid"
  >
    <div
      className="column"
    />
    <div
      className="column"
    >
      <div
        className="ui placeholder"
      >
        <div
          className="rectangular image"
        />
      </div>
    </div>
    <div
      className="column"
    >
      <div
        className="ui placeholder"
      >
        <div
          className="line"
        />
      </div>
    </div>
    <div
      className="column"
    />
  </div>,
  <div
    className="ui divider"
  />,
  <div
    className="ui four column grid"
  >
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
  </div>,
]
`;

exports[`PriestProfileContainer element should render an error message when showError and loading are both true 1`] = `
<div
  className="ui container"
  style={
    Object {
      "marginTop": "7em",
    }
  }
>
  <h2
    className="ui header red"
  >
    Could not estabilish server connection.
  </h2>
  <h3
    className="ui header"
  >
    <a
      aria-current="page"
      className="hoverLink active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      Back to the Home Page
    </a>
  </h3>
</div>
`;

exports[`PriestProfileContainer element should render an error message when showError is true 1`] = `
<div
  className="ui container"
  style={
    Object {
      "marginTop": "7em",
    }
  }
>
  <h2
    className="ui header red"
  >
    Could not estabilish server connection.
  </h2>
  <h3
    className="ui header"
  >
    <a
      aria-current="page"
      className="hoverLink active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      Back to the Home Page
    </a>
  </h3>
</div>
`;

exports[`PriestProfileContainer element should render an error message when showError is true and editMode=true 1`] = `
<div
  className="ui container"
  style={
    Object {
      "marginTop": "7em",
    }
  }
>
  <h2
    className="ui header red"
  >
    Could not estabilish server connection.
  </h2>
  <h3
    className="ui header"
  >
    <a
      aria-current="page"
      className="hoverLink active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      Back to the Home Page
    </a>
  </h3>
</div>
`;
